### PR TITLE
chore(PanoViewer): isWebGLAvailable spec change

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue

Ref #243

## Details

isWebGLAvailable should return true if PanoViewer is avilable.

### CODE REVIEW

File: src/PanoViewer/PanoViewer.js [hunk 1]
```diff
@@ -907,7 +907,7 @@ export default class PanoViewer extends Component {
 	 * @static
 	 */
 	static isWebGLAvailable() {
-		return WebGLUtils.isWebGLAvailable();
+		return WebGLUtils.isWebGLAvailable() && WebGLUtils.isStableWebGL();
```
Comment (younkue) [inline src/PanoViewer/PanoViewer.js]: @jongmoon  Is isStableWebGL just webgl, not experiment-webgl?

Comment (jongmoon) [reply src/PanoViewer/PanoViewer.js]: - `isWebGLAvailable`: just webgl
  - `isStableWebGL`: check if above android 4.4 chrome

Comment (younkue) [reply src/PanoViewer/PanoViewer.js]: @jongmoon Is isWebGLAvailable in the constructor OK?

Comment (jongmoon) [reply src/PanoViewer/PanoViewer.js]: @jongmoon Of course~

Comment (younkue) [reply src/PanoViewer/PanoViewer.js]: @jongmoon LGTM~!

Comment (jongmoon) [reply src/PanoViewer/PanoViewer.js]: @younkue There would be change for this function. This function will be restored and `isSupported` function will be added to provide `WebGLUtils.isWebGLAvailable() && WebGLUtils.isStableWebGL();`

Comment (younkue) [reply src/PanoViewer/PanoViewer.js]: @jongmoon  ```isSupported``` is much more intuitive and popular.

Comment (younkue) [reply src/PanoViewer/PanoViewer.js]: @jongmoon  Do you use isWebGLAvailable?

**Pull request merged**